small3/smlgp3.g

```diff
@@ -351,7 +351,7 @@ end;
 ##                  
 SELECT_SMALL_GROUPS_FUNCS[ 11 ] := function( size, funcs, vals, inforec, all,
                                              id, idList)
-    local result, i, g, ok, j, range;
+    local result, i, g, ok, j, range,nid;
```

Comment:
Sorry for a nitpick, but missing space after the comma before the new identifier ...